docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     volumes:
       - 'redis:/data'
   web:
+    image: spree_starter
```

Comment:
@r3cha I think you're missing a declaration for this image, or I'm missing something?

Reply:
@damianlegawiec Thanks, yes you right